openfisca_core/variables.py

```diff
@@ -166,9 +171,19 @@ def __init__(self, baseline_variable = None):
         self.calculate_output = self.set_calculate_output(attr.pop('calculate_output', None))
         self.is_period_size_independent = self.set(attr, 'is_period_size_independent', allowed_type = bool, default = VALUE_TYPES[self.value_type]['is_period_size_independent'])
         self.base_function = self.set_base_function(attr.pop('base_function', None))
-        self.formula = Formula.build_formula_class(attr, self, baseline_variable)
+
+        formulas_attr, other_attrs = _partition(attr, lambda name, value: name.startswith(FORMULA_NAME_PREFIX))
```

Comment:
Why define `FORMULA_NAME_PREFIX`at the top of the page and not right befor this line ? (To make it easier for readers to find the value)